pkg/arvo/sys/vane/dill.hoon

```diff
@@ -100,8 +105,11 @@
               gift:able:ames                            ::
       ==  ==                                            ::
       $:  %b                                            ::
-          $%  $>(%mass gift:able:behn)                  ::
-              $>(%writ gift:able:clay)                  ::  XX %slip
+          $%  $>  $?  %mass                             ::
+                      %wake                             ::
+                      %writ                             ::  XX %slip
+                  ==                                    ::
+              gift:able:behn                            ::
```

Comment:
This refactoring is wrong -- my bad.

Reply:
It still works fine in normal operation, but fails boot. Looks like `$>` needs to assert that the tag actually exists somehow.

Reply:
I guess it does when given a single tag, but not when combined with `$?`. Not sure if this pattern is righteous ....